beta-src/src/components/ui/WDPhaseUI.tsx

```diff
@@ -14,13 +14,22 @@ import {
 import WDCountdownPill from "./WDCountdownPill";
 import WDPillScroller from "./WDPillScroller";
 
+const getCurPhaseMinutes = function (phaseMinutes, phaseMinutesRB, phase) {
```

Comment:
Same comment as earlier about `get` - I think these are selectors or renderers, and would be better without the `get` prefix.